src/components/Layout/Header/NavBar/SubmenuHeader.tsx

```diff
@@ -21,7 +22,7 @@ export const SubmenuHeader = ({ title, description }: ExpandedMenuItemProps) =>
         <IconButton
           isRound
           size='sm'
-          onClick={handleBackClick}
+          onClick={onBackClick ?? handleBackClick}
```

Comment:
Once the above changes are made:

```suggestion
          onClick={handleBackClick ?? handleBackClickDefault}
```